android/java/res/xml/brave_privacy_preferences.xml

```diff
@@ -168,7 +168,7 @@
             android:order="25"
             android:title="@string/sns_title" />
 
-        <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
+        <!-- <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
```

Comment:
If we moved these to other place, this should be just deleted.

Reply:
sorry it's my bad. updating now.